client/pkg/template/order.go

```diff
@@ -16,10 +16,11 @@ import (
 
 // Canonical order of resources in the generated list.
 var canonicalResourceOrder = map[resource.Type]int{
-	omni.ClusterType:        1,
-	omni.ConfigPatchType:    2,
-	omni.MachineSetType:     3,
-	omni.MachineSetNodeType: 4,
+	omni.ClusterType:                 1,
+	omni.ExtensionsConfigurationType: 2,
```

Comment:
would this affect for an existing cluster going from no extensions to extensions?